src/webgpu/web_platform/external_texture/video.spec.ts

```diff
@@ -4,6 +4,7 @@ Tests for external textures from HTMLVideoElement (and other video-type sources?
 - videos with various encodings, color spaces, metadata
 
 TODO: consider whether external_texture and copyToTexture video tests should be in the same file
+TODO: add a test configuration for red-green.mp4 to the importExternalTexture Sample test once the infrastructure can load it succesfully.
```

Comment:
What infrastructure can't load it?

If it's in Chromium, we should just add the test and add a suppression in Dawn's `expectations.txt`. (Though this can be a separate PR if it's nontrivial.)

Reply:
Removed the TODO. Not sure if this is a cts issue or a Chromium issue, but I've noted it in a bug here:  https://bugs.chromium.org/p/dawn/issues/detail?id=1432

At the least, running it the standalone cts with Chromium will always fail to load. It hits this error: https://github.com/gpuweb/cts/blob/main/src/webgpu/web_platform/util.ts#L36

It's already disabled in chromium here: https://source.chromium.org/chromium/chromium/src/+/main:content/test/gpu/gpu_tests/test_expectations/webgpu_cts_expectation_queries.js;l=1618

Note that all external texture tests have been disabled in Chromium for awhile for some unknown reason, which I intend to turn back on soon here: https://source.chromium.org/chromium/chromium/src/+/main:third_party/dawn/webgpu-cts/expectations.txt;l=242?q=expectations.txt

Reply:
Ah, cool, I see now the mp4 was already there. I think the parameters match now so the existing expectation should still work. (A dry run will double check this, too.)